activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+*   Change `ActiveSupport::Duration` creation with numeric methods like `1.week`
+    to create durations with more predictable and ISO8601-conformant parts.
+
+    This is to remove astonishment from getting `3600 seconds` from `1.hour`.
+
+    It should not affect current apps as duration's `value` (number of seconds) remains the same,
+    only hash of `parts` (and `inspect` value) is changed and only when it's constructed by calling
+    methods on integers. Manual construction of Durations isn't affected.
+    Inside the ActiveSupport::Duration itself most operations rely only on number of seconds.
```

Comment:
@prathamesh-sonpatki It's important for the changelog because it answers Rails users' primary question: will this break compatibility for my app / library / plugin?